File: refactoring-practice-1/src/main/java/dojo/inline_class/Phone.java
package dojo.inline_class;

public class Phone {

	private String areaCode;
	private String number;
	
	public String getTelephoneNumber() {
		return "(" + areaCode + ") " + number;
	}

	public String getAreaCode() {
		return areaCode;
	}

	public void setAreaCode(String areaCode) {
		this.areaCode = areaCode;
	}

	public String getNumber() {
		return number;
	}

	public void setNumber(String number) {
		this.number = number;
	}
	
	
}

File: refactoring-practice-1/src/main/typescript/inline_class/phone.ts
export class Phone {

    private areaCode: string;
    private number: string;

    public getTelephoneNumber() {
        return "(" + this.areaCode + ") " + this.number;
    }

    public getAreaCode() {
        return this.areaCode;
    }

    public setAreaCode(areaCode: string) {
        this.areaCode = areaCode;
    }

    public getNumber() {
        return this.number;
    }

    public setNumber(number: string) {
        this.number = number;
    }


}
